doc/en/user/source/data/app-schema/solr-tutorial.rst

```diff
@@ -93,4 +93,115 @@ The complete mapping file is:
       </typeMappings>
   </as:AppSchemaDataAccess>
 
-The mappings for the attributes are straightforward and follow the normal App-Schema attributes mappings syntax. Currently multi valued fields are not supported.
\ No newline at end of file
+The mappings for the attributes are straightforward and follow the normal App-Schema attributes mappings syntax. Currently multi valued fields are not supported.
+
+Using Solr as App-Schema Indexes
+--------------------------------
+
+App-Schema Indexes is an extension for mapping that allows to use Apache Solr as Index for queries and retrieving data from normal App-Schema datasource (SQL DB, MongoDB, ... ).
+
+The only requirement to use it is having Geoserver App-Schema extension and Solr extension installed.
+
+How Index layer works
+^^^^^^^^^^^^^^^^^^^^^
+
+When App-Schema detects the index layer is activated for a FeatureType, it will use Solr configured fields for every query incoming from Geoserver OWS requests.  If the incoming query uses only indexed fields App-Schema will query only on Solr data source for retrieving matching features IDs and will connect to normal data source to get all in depth data but exclusively for matching IDs.
```

Comment:
This text needs to be reviewed, I would use an example use case to explain this.